docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md

```diff
@@ -7,6 +7,12 @@ in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.9.1] - 2023-08-25
```

Comment:
```suggestion
## [1.10.0] - 2023-08-25
```

Adding new features in a backwards compatible manner requires us to bump the minor version instead of the patch version which is reserved for bug fixes in a backwards compatible manner.

Reply:
Ah yep NP will fix :+1: